apps/aecontract/test/aecontract_SUITE.erl

```diff
@@ -3472,7 +3472,9 @@ sophia_signatures_aens(Cfg) ->
     Acc             = ?call(new_account, 20000000 * aec_test_utils:min_gas_price()),
     NameAcc         = ?call(new_account, 20000000 * aec_test_utils:min_gas_price()),
     Ct              = ?call(create_contract, NameAcc, aens, {}, #{ amount => 100000 }),
-    Name1           = <<"bla.test">>,
+    LongPrefix      = <<"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx">>,
```

Comment:
Why are we replacing the existing test with this one?

Reply:
@ThomasArts In some cases of old test we just want to use registered name. The Long Prefix falls back to non-auction behaviour (it's over 32 characters). Surly, like that it is for state channel test. Here, I think, we might want to have new test with auction and then with short name.